doc/en/index.md

```diff
@@ -0,0 +1,60 @@
+English / [Russian](doc/ru/index.md)
+
+# RHVoice documentation
+
+<details>
+<summary>
+About the Speech synthesizer
+</summary>
+
+## Features
+
+### Speech synthesis method
+
+RHVoice uses statistical parametric synthesis. It relies on existing open source
+speech technologies (mainly [HTS](http://hts.sp.nitech.ac.jp) and related
+software). 
+
+Voices are built from recordings of natural speech. They have small footprints,
+because only statistical models are stored on users' computers. And though the
+voices lack the naturalness of the synthesizers which generate speech by
+combining segments of the recordings themselves, they are still very

```

Comment:
May make sense to mention RNN-based synhesizers.

Reply:
No. It isn't readme.md. It's short information.